scripts/install_third_party.sh

```diff
@@ -117,6 +117,25 @@ fi
 echo Checking if numpy is installed in $TOOLS_DIR/pip_packages
 if [ ! -d "$TOOLS_DIR/numpy-1.6.1" ]; then
   echo Installing numpy
+
+  if ! type pip > /dev/null 2>&1 ; then
+      echo ""
+      echo "  Numpy is required for Oppia to operate properly, but pip wasn't found"
+      echo "  on your local machine."
+      echo ""
+      echo "  Please see \"Installing Oppia\" on the Oppia developers' wiki page:"
+
+      if [ "${OS}" == "Darwin" ] ; then
+        echo "  https://github.com/oppia/oppia/wiki/Installing-Oppia-%28Mac-OS%29"
+      elif [ "${OS}" == "Linux" ] ; then
+        echo "  https://github.com/oppia/oppia/wiki/Installing-Oppia-%28Linux%29"
+      else
```

Comment:
Ok, so lines with `elif` will go away, and link for Linux url will replace Windows url